### PR TITLE
Set props path in ConfigOpts. Fixes #676

### LINKED ISSUE #676: Seeing NPE in rfile-info in 2.0.0-SNAP
Seeing the following NPE in master when running rfile-info. Running accumulo built from 
caf75c15deffe1e0041d2242a350988a17641e58 with local changes from #675

```
$ accumulo rfile-info /accumulo/tables/3/t-000005l/A00000am.rf
Reading file: hdfs://localhost:8020/accumulo/tables/3/t-000005l/A00000am.rf
No on disk encryption detected.
RFile Version            : 8

Locality group           : <DEFAULT>
	Num   blocks           : 2,172
	Index level 0          : 68,056 bytes  1 blocks
	First key              : 73333378a7c00468 0921:707c [] 1538599801516 false
	Last key               : 79999982c534792e 78c2:67e0 [] 1538599767359 false
	Num entries            : 2,018,247
	Column families        : <UNKNOWN>

2018-10-03 17:06:31,101 [start.Main] ERROR: Thread 'rfile-info' died.
java.lang.NullPointerException
	at com.beust.jcommander.JCommander.expandArgs(JCommander.java:430)
	at com.beust.jcommander.JCommander.parse(JCommander.java:340)
	at com.beust.jcommander.JCommander.parse(JCommander.java:319)
	at org.apache.accumulo.core.cli.Help.parseArgs(Help.java:34)
	at org.apache.accumulo.core.cli.ConfigOpts.parseArgs(ConfigOpts.java:93)
	at org.apache.accumulo.core.file.rfile.bcfile.PrintInfo.main(PrintInfo.java:73)
	at org.apache.accumulo.core.file.rfile.PrintInfo.execute(PrintInfo.java:187)
	at org.apache.accumulo.start.Main.lambda$execKeyword$0(Main.java:129)
	at java.lang.Thread.run(Thread.java:748)
```

### PR DESCRIPTION
**Pull request by milleruntime**
We are getting the properties path after calling getSiteconfiguration but it was never set so a null is getting passed to JCommander.

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java [hunk 1]
```diff
@@ -40,6 +41,10 @@
   private String propsPath;
 
   public String getPropertiesPath() {
+    if (propsPath == null) {
```
Comment (keith-turner) [inline core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java]: could sync this method.

File: core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java [hunk 2]
```diff
@@ -39,7 +40,11 @@
       + "The classpath will be searched if this property is not set")
   private String propsPath;
 
-  public String getPropertiesPath() {
+  public synchronized String getPropertiesPath() {
+    if (propsPath == null) {
+      URL propLocation = SiteConfiguration.getAccumuloPropsLocation();
```
Comment (mikewalch) [inline core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java]: Could this just be one line?

Comment (mikewalch) [reply core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java]:
```java
propsPath = SiteConfiguration.getAccumuloPropsLocation().getFile();

Comment (milleruntime) [reply core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java]: Yeah

**Pull request merged**